### PR TITLE
[wasm][debugger] Add support for invoking getters on ValueTypes

### PR DESCRIPTION
**Pull request by radical**
- this allows IDEs to let users invoke property getters for ValueTypes

### CODE REVIEW

**Comment by lewing** [comment]
@radical this is going to need a merge

File: src/mono/wasm/debugger/DebuggerTestSuite/Support.cs [hunk 1]
```diff
@@ -301,54 +302,31 @@ internal void CheckNumber<T>(JToken locals, string name, T value)
...
-                else
-                {
-                    Assert.Equal("string", val["type"]?.Value<string>());
-                    Assert.Equal(value, val["value"]?.Value<string>());
-                }
-                return;
-            }
-            Assert.True(false, $"Could not find variable '{name}'");
+            var l = GetAndAssertObjectWithName(locals, name);
+            CheckValue(l["value"], TString(value), name).Wait();
```
Comment (lewing) [inline src/mono/wasm/debugger/DebuggerTestSuite/Support.cs:314]: This appears to break things because TString assumes description is set.

**Review by lewing — CHANGES_REQUESTED** [review]

File: src/mono/wasm/debugger/DebuggerTestSuite/Support.cs (hunk 1 above)
Comment (radical) [reply src/mono/wasm/debugger/DebuggerTestSuite/Support.cs:314]: Break where?

Comment (lewing) [reply src/mono/wasm/debugger/DebuggerTestSuite/Support.cs:314]: JSExceptionTestAll and ExceptionTestUncaught

Comment (lewing) [reply src/mono/wasm/debugger/DebuggerTestSuite/Support.cs:314]:
```
  Error Message:
   System.AggregateException : One or more errors occurred. (One or more errors occurred. ([message] Could not find value field named description
Expected: True
Actual:   False))
---- System.AggregateException : One or more errors occurred. ([message] Could not find value field named description
Expected: True
Actual:   False)
-------- [message] Could not find value field named description
Expected: True
Actual:   False
  Stack Trace:
```

Comment (lewing) [reply src/mono/wasm/debugger/DebuggerTestSuite/Support.cs:314]: I removed the description check, we should double check but I don't think the frontends actually use the decscription field on string at all.

**Review by lewing — APPROVED** [review]
There are some formatting issues remaining but I think I'll do a dotnet-format run on all the debugger code as a follow-up

**Comment by lewing** [comment]
This change is wasm specfic and can't be causing the OSX failure.

**Pull request merged**